source/calculus/source/03-AD/08.ptx

```diff
@@ -199,7 +433,30 @@
 
         <figure xml:id="F-3-4-PA1" permid="RHW">
           <caption>A rectangular parcel with a square end.</caption>
-          <image width="47%" source="3_4_PA1" />
+          <image width="47%">
+            <sageplot>
+              from sage.plot.graphics import Graphics
+              front = polygon2d([(0,0), (0,2), (2,2), (2,0)],fill=True, color='purple', alpha = 0.5)
+              front_border=polygon2d([(0,0), (0,2), (2,2), (2,0)], fill=False,color='black',thickness=2)
+              back = polygon2d([(4,2), (4,4), (6,4),(6,2)], fill = True, color='purple')
+              back_border= polygon2d([(4,2), (4,4), (6,4),(6,2)], fill = False, color='black', thickness=2)
+              left = polygon2d([(0,0),(0,2),(4,4),(4,2)], fill=True, color='purple')
+              left_border = polygon2d([(0,0),(0,2),(4,4),(4,2)], fill=False, color='black', thickness=2)
+              right = polygon2d([(2,0), (2,2), (6,4),(6,2)],fill=True, color='purple', alpha = 0.5)
+              right_border = polygon2d([(2,0), (2,2), (6,4),(6,2)],fill=False, color='black', thickness=2)
+              geometry = Graphics()
+              geometry += left
+              geometry += left_border
+              geometry += front
+              geometry += front_border
+              geometry += back
+              geometry += back_border
+              geometry += right
+              geometry += right_border
+              geometry.axes(False)
+              geometry
+            </sageplot>
+          </image>
```

Comment:
Old:

![image](https://github.com/user-attachments/assets/ecc9fa73-b1d7-440a-8d5f-3cc1d438f730)

New:

![image](https://github.com/user-attachments/assets/91b9e69e-238e-4b5a-964b-e9327b9c027c)

Maybe it's just me, but I think its easier to understand the image when the darker side is on bottom.

Reply:
If possible, using dashes for sides obscured to the viewer might help as well.

Reply:
the most recent commit changes the shading (and doesn't remove the helper axes, oops), and i'm working on adding the dashed lines